lmdeploy/disagg/README.md

```diff
@@ -0,0 +1,58 @@
+# LMDeploy-DistServe
+
+## Key Components
+1. ​**Router Service**: Coordinates between prefill/decode engines
+4. ​**Migration Manager**: Facilitates high-performance memory sharing
+
+## Installation
+```
+# Inference Engine
+pip install lmdeploy[all] >= 0.7.0
+
+# Transfer Engine
```

Comment:
Just curious. Is it a private Python package for transferring? I couldn't find any information about this project. Thanks.

Reply:
https://github.com/JimyMa/DLSlime